src/sidebar/util/test/sentry-test.js

```diff
@@ -75,6 +75,20 @@ describe('sidebar/util/sentry', () => {
       );
     });
 
+    it('configures Sentry to ignore Errors with matching the text "Fetch operation failed"', () => {
```

Comment:
These tests are a little...less than totally useful, but I'm happy to play along!